ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/passwordrecovery/PasswordRecoveryFormController.java

```diff
@@ -127,10 +137,21 @@ public String generateToken(
 		}
 		
 		try {
-			// Finds the user using the email as key, if it exists a new token is generated to include in the unique http URL.
 			Account account = this.accountDao.findByEmail(formBean.getEmail());
+			List<Group> group = this.groupDao.findAllForUser(account.getUid());
+			// Finds the user using the email as key, if it exists a new token is generated to include in the unique http URL.
+			
+
+			
+			for (Group g : group) {
+				if (g.getName().equals("PENDING_USERS")) {
```

Comment:
The group was just renamed to "PENDING" in master ... (last week)
Sorry about that ;-)